src/styles/Grid.module.scss

```diff
@@ -0,0 +1,123 @@
+.grid-images [class*='#{$prefix}--col'] {
+    @include carbon--breakpoint-down('md') {
+      padding-right: 0px;
+    }
+  }
+
+// gallery clusters
+.mock-gallery {
+    @include carbon--breakpoint-down('md') {
+      --space: 0;
+    }
+  }
+  
+  .mock-gallery [class*='#{$prefix}--col'] {
+    @include carbon--breakpoint-down('md') {
+      margin-top: 1.5rem;
+    }
+  }
+
+// grid classnames (should be added to either grid package or theme?)
```

Comment:
I thought all of these no gutter styles were already in the theme?

Reply:
okay it actually seams these styles weren't doing anything...so just deleted them for now.